api/v1beta1/flowcollector_types.go

```diff
@@ -318,8 +318,8 @@ type FLPMetrics struct {
 	Server MetricsServerConfig `json:"server,omitempty"`
 
 	// `ignoreTags` is a list of tags to specify which metrics to ignore. Each metric is associated with a list of tags. More details in https://github.com/netobserv/network-observability-operator/tree/main/controllers/flowlogspipeline/metrics_definitions .
-	// Available tags are: `egress`, `ingress`, `flows`, `bytes`, `packets`, `namespaces`, `nodes`, `workloads`.
-	//+kubebuilder:default:={"egress","packets"}
+	// Available tags are: `egress`, `ingress`, `flows`, `bytes`, `packets`, `namespaces`, `nodes`, `workloads`, `health`.
+	//+kubebuilder:default:={"egress","packets", "health"}
```

Comment:
this would ignore health by default, this isn't what we want?

Reply:
Maybe a better naming can be found ?

The metrics flagged with `health` are only flows `metrics definitions` per nodes / namespaces / workloads

The health dashboard will look like as before by default

Reply:
(or health is not what I thought it is )

Reply:
It seems like, at least for now, "health" fully overlaps with "flows" so we could keep just "flows"